src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -231,11 +247,15 @@ void ClusterTaskManager::DispatchScheduledTasksToWorkers(
           // double-acquiring when the next invocation of this function tries to schedule
           // this task.
           cluster_resource_scheduler_->ReleaseWorkerResources(allocated_instances);
-          // No worker available, we won't be able to schedule any kind of task.
-          // Worker processes spin up pretty quickly, so it's not worth trying to spill
-          // this task.
           ReleaseTaskArgs(task_id);
-          return;
```

Comment:
I just realized this is a `return` not a break. Thinking about how to replicate this behavior now